src/index.html

```diff
@@ -20,8 +20,373 @@
+          alt="imac"
+          class="card__img"
+        />
+
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__productCode">Product code: 195434</p>
+
+        <div class="card__review-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price_label">Price:</span>
+          <span class="card__price_amount">$2,199</span>
+        </div>
+
+        <button
+          type="button"
+          class="card__buyButton"
+        >
+          BUY
+        </button>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__productCode">Product code: 195434</p>
+
+        <div class="card__review-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price_label">Price:</span>
+          <span class="card__price_amount">$2,199</span>
+        </div>
+
+        <button
+          type="button"
+          class="card__buyButton"
+        >
+          BUY
+        </button>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__productCode">Product code: 195434</p>
+
+        <div class="card__review-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price_label">Price:</span>
+          <span class="card__price_amount">$2,199</span>
+        </div>
+
+        <button
+          type="button"
+          class="card__buyButton"
+        >
+          BUY
+        </button>
```

Comment:
Each card seems to be duplicated with the same content. If these cards are meant to be the same, consider using a template engine or a loop in your JavaScript to generate them dynamically. This will make your code DRY (Don't Repeat Yourself) and easier to maintain.